libraries/AP_RangeFinder/AP_RangeFinder_TeraRangerDUO.cpp

```diff
@@ -0,0 +1,142 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
```

Comment:
If you write the following device specifications and URL of video, it will be easier to maintain.
Other device drivers also describe URLs such as specifications.

Web: http://www.teraranger.com/product/teraranger-duo/
Manual: https://www.terabee.com/wp-content/uploads/2017/09/TR_Duo_Manual_FW4.0.0_DUO-1.pdf
TestVideo with precise mode: https://www.youtube.com/watch?v=mnKjZbB9USM

Reply:
Agree.